packages/controllers/organization_form.coffee

```diff
@@ -1,26 +1,39 @@
 if Meteor.isClient
+  Template.organizationForm.onCreated ->
+    organization = new Organization()
+    @organization = new ReactiveVar(organization)
+
+  Template.organizationForm.helpers
+    errorForName: ->
+      Template.instance().organization.get().getValidationErrors().name
```

Comment:
@aslagle this helper doesn't get run again if the user changes the name so that there are no longer any errors. I'm reading about ReactiveVar now and it seems like this should recalculate when the organization gets updated?  Any idea what's going on here?